index.html

```diff
@@ -45,30 +45,76 @@ <h1>{A little bit about me...}</h1>
 				<p>Please keep scrolling to see some of my work.</p>
```

Comment:
on line 28 put the href inside an anchor tag inside the button tag, not sure you can have a href inside a button tag